packages/client/tsconfig.test.json

```diff
@@ -1,6 +1,8 @@
 {
   "extends": "./tsconfig.json",
   "compilerOptions": {
+    "module": "es2020",
+    "target": "es2020",
```

Comment:
This is redundant with the configuration in extended config.

Reply:
Removed.